lib/elixir/lib/kernel.ex

```diff
@@ -5753,6 +5753,69 @@ defmodule Kernel do
     end
   end
 
+  @doc """
+  Debugs the given `code`.
+
+  `dbg/2` can be used to debug the given `code` through a configurable debug function.
+
+  ## Configuring debug function
+
+  The debug function can be configured at compile time through the `:dbg_fun` key
```

Comment:
We typically don't use `_type` suffixes in our configuration. So I would go with `dbg_backend` or `dbg_callback`. If we want to keep the type, I would go with `dbg_mfa`.

Reply:
Got it 👍 Done.